moveit_setup_assistant/moveit_setup_core_plugins/src/start_screen_widget.cpp

```diff
@@ -305,8 +305,16 @@ void StartScreenWidget::onPackagePathChanged(const QString& /*path*/)
 
 void StartScreenWidget::onUrdfPathChanged(const QString& path)
 {
-  setup_step_.loadURDFFile(path.toStdString(), urdf_file_->getArgs().toStdString());
-  urdf_file_->setArgsEnabled(setup_step_.isXacroFile());
+  try
+  {
+    setup_step_.loadURDFFile(path.toStdString(), urdf_file_->getArgs().toStdString());
+    urdf_file_->setArgsEnabled(setup_step_.isXacroFile());
+  }
+  catch (const std::runtime_error& e)
+  {
+    RCLCPP_ERROR(setup_step_.getLogger(), "%s", e.what());
```

Comment:
Should we also add the "Error Loading URDF path" prefix to the `RCLCPP_ERROR` macro for consistency?